spec/features/new_education_experience_spec.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'Add new Education Experience', type: :feature do
+  let!(:authed_user) { create_logged_in_user }
+
+  it 'displays the new education experience message' do
+    visit '/education_experiences/new'
+
+    expect(page).to have_content ('Nova experiência educacional')
+  end
+
+  it 'has the a link to create a new user experience' do
+    visit '/education_experiences'
+
+    expect(page).to have_link(t('education_experiences.new.new_experience'))
```

Comment:
use the text directly in the test, this way if the translation has a typo or is wrong, is easier to notice